src/humancellatlas/data/metadata/helpers/staging_area.py

```diff
@@ -215,40 +215,39 @@ def _entity_ids_by_type(self,
 
 @attr.s(frozen=True, kw_only=True, auto_attribs=True)
 class CannedStagingAreaFactory:
-    #: The local path to the repository
-    local_path: Path
+    #: A local path containing one or more staging areas
+    base_path: Path
 
     @classmethod
     def clone_remote(cls, remote_url: furl, local_path: Path, ref: str) -> Self:
         """
         Clone a Git repository into a local path.
 
-        :param remote_url: A GitHub repository clone URL (HTTPS)
+        :param remote_url: A valid Git url
```

Comment:
If you say "valid" here, you'd have to say "valid" everywhere because hardly any of our code accepts both valid and invalid things.

And I don't know what a "Git url" is.